simpegBranchInfo.txt

```diff
@@ -0,0 +1,41 @@
+https://github.com/simpeg/simpeg.git
```

Comment:
I don't know either, came with merging from dev -> em/dev -> mt/dev.  I don't think this is my doing.